Docs/source/gravity.rst

```diff
@@ -447,7 +447,7 @@ option works by adding the gravitational acceleration of the point
 mass onto the acceleration from whatever other gravity type is under
 in the simulation.
 
-.. note:: the point mass have a mass < 0
+.. note:: The point mass may have a mass < 0
```

Comment:
As far as I can tell this is actually true, but it worries me that it's true.

Reply:
me too